pkgs/tools/system/lanzaboote/tool.nix

```diff
@@ -0,0 +1,56 @@
+{ systemd
+, makeWrapper
+, binutils-unwrapped
+, sbsigntool
+, rustPlatform
+, fetchFromGitHub
+, lib
+, uefiPkgs
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "lanzaboote_tool";
+  version = "0.3.0";
+
+  src = fetchFromGitHub {
+    owner = "nix-community";
+    repo = "lanzaboote";
+    rev = "v${version}";
+    hash = "sha256-Fb5TeRTdvUlo/5Yi2d+FC8a6KoRLk2h1VE0/peMhWPs=";
+  };
+
+  sourceRoot = "source/rust/tool";
+  cargoHash = "sha256-ebaLnq2takCEvbHVqkI73FVTwjn1u80nnuy+GzVnROA=";
+
+  env.TEST_SYSTEMD = systemd;
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  postInstall = ''
+    # Clean PATH to only contain what we need to do objcopy. Also
+    # tell lanzatool where to find our UEFI binaries.
+    mv $out/bin/lzbt $out/bin/lzbt-unwrapped
+    makeWrapper $out/bin/lzbt-unwrapped $out/bin/lzbt \
+      --set PATH ${lib.makeBinPath [ binutils-unwrapped sbsigntool ]} \
+      --set LANZABOOTE_STUB ${passthru.stub}/bin/lanzaboote_stub.efi
+  '';
+
+  nativeCheckInputs = [
+    binutils-unwrapped
+    sbsigntool
+  ];
+
+  passthru = {
+    stub = uefiPkgs.lanzaboote-stub;
+  };
```

Comment:
It's not worth it to do all the ad-hoc bootstrapping stuff just for a `passthru`. Sever this, and leave it to the NixOS module to correctly combine the correct `lanzboot` and `lanzaboot-stub` versions together.

Reply:
This will cause semi-wrapping or leak of how the build should happen in the NixOS module (if you do no wrapping at all in the package definition), I suppose what you are saying is that either case is fine.